qa/tasks/cephfs/test_exports.py

```diff
@@ -114,11 +117,19 @@ def test_export_pin(self):
             self._wait_subtrees(status, 0, [('/1', 0), ('/1/4/5', 1), ('/a', 1), ('/a/b/aa/bb', 0)])
 
         # Test getfattr
-        self.assertTrue(self.mount_a.getfattr("1", "ceph.dir.pin") == "0")
-        self.assertTrue(self.mount_a.getfattr("1/4", "ceph.dir.pin") == "-1")
-        self.assertTrue(self.mount_a.getfattr("1/4/5", "ceph.dir.pin") == "1")
-        if (len(self.fs.get_active_names()) > 2):
-            self.assertTrue(self.mount_a.getfattr("1/2/3", "ceph.dir.pin") == "2")
+        def test_getfattr():
+            if isinstance(self.mount_a, KernelMount):
+                p = self.mount_a.client_remote.run(args=['uname', '-r'], stdout=StringIO(), wait=True)
+                dir_pin = self.mount_a.getfattr("1", "ceph.dir.pin")
+                log.debug("mount.getfattr('1','ceph.dir.pin'): %s " % dir_pin)
+	        if str(p.stdout.getvalue()) < "5" and not(dir_pin):
+	            raise SkipTest("Require FUSE client")
```

Comment:
After reading the docs while trying to understand the failure [1], I learned that we should be calling `self.skipTest` instead. I opened #27761 to fix the other instances of this. Please change your PR too.

As for the failure itself, I'm stumped. Let's see if the `self.skipTest` change does anything. Also, the message would be better as `self.skipTest("kernel does not support getting ceph.dir.pin")`.

[1] http://qa-proxy.ceph.com/teuthology/sidharthanup-2019-04-23_14:49:59-multimds-master-distro-default-smithi/3882861/teuthology.log